src/client/client.cpp

```diff
@@ -477,7 +477,8 @@ void Client::step(float dtime)
 		if (envEvent.type == CEE_PLAYER_DAMAGE) {
 			u16 damage = envEvent.player_damage.amount;
 
-			if (envEvent.player_damage.send_to_server && ! g_settings->getBool("prevent_natural_damage"))
+                        //if (envEvent.player_damage.send_to_server && ! g_settings->getBool("prevent_natural_damage"))
```

Comment:
Please remove the comment.

Reply:
Also, if (envEvent.player_damage.send_to_server) should stay as it's from minetest code and not a dragonfire addition.

Reply:
mmh yeah i'll test those changes in a bit. I just more or less copied my original noFallDamage hack which was admittedly done with just about 0 knowledge of the codebase. We didn't find any regressions doing it this way tho. But yeah i'll have a look

Reply:
Well what can I say you're 100% right on both accounts .. this one should have the comment removed as you a said and the other one should say settings && damage !=0 sadly i cant seem to be able to commit anything to this rn .. github has somehow changed what repo i forked this from ... man this stuff is annoying lol